src/sass/components/_tour.scss

```diff
@@ -37,14 +37,73 @@
 
 .popover-wrapper {
   position: absolute;
-  top: -10px;
+  left: -($spacing-xx-small);
+  top: -($spacing-small);
+
+  .epic-progress & {
+    top: -($spacing-medium);
+  }
 
   .logo-popover & {
-    top: unset;
+    top: $spacing-small;
+    left: $spacing-large;
+  }
```

Comment:
@dvdherron If we target `.logo-popover > .popover-wrapper` then we don't need to also include the override for `.user-info &`

Also, these 2 properties can be re-ordered by alpha.

Reply:
done. didn't need that override there.